src/guides/v2.4/install-gde/prereq/php-settings.md

```diff
@@ -73,14 +73,23 @@ PHP Warning:  date(): It is not safe to rely on the system's timezone settings.
       -  Installing and updating Magento components from Magento Marketplace, `2G`
```

Comment:
This can probably be removed now that the web set up wizard was removed in Magento 2.4.0?